ena-submission/environment.yml

```diff
@@ -17,3 +17,5 @@ dependencies:
   - xmltodict
   - biopython
   - pytz
+  - pip:
+    - tqdm-loggable
```

Comment:
Does this dependency look stable enough to add? From the github repo it doesn't look like its that actively maintained: https://github.com/tradingstrategy-ai/tqdm-loggable